scenarios/kubernetes_e2e.py

```diff
@@ -234,6 +234,7 @@ def add_environment(self, *envs):
             'GOPATH',
             'GOROOT',
             'HOME',
+            'KUBEKINS_TIMEOUT',  # Use --timeout
```

Comment:
this is docker-mode only. I would suggest the same as https://github.com/kubernetes/test-infra/pull/2833#discussion_r118083089